platforms/unix/vm/sqUnixMain.c

```diff
@@ -1467,7 +1464,8 @@ static int vm_parseArgument(int argc, char **argv)
 
   /* vm arguments */
 
-  if      (!strcmp(argv[0], VMOPTION("help")))		{ usage();		return 1; }
+  if      (!strcmp(argv[0], VMOPTION("help")))		{ usage(0);         /*NOTREACHED*/}
+  else if (!strcmp(argv[0], VMOPTION("version")))	{ versionInfo();	return 0; }
```

Comment:
I just relocated it to keep it closer to help, so it is easy to see that help/version are valid options that exit with 0.